logstash-core/spec/support/helpers.rb

```diff
@@ -77,7 +78,9 @@ def start_agent(agent)
     end
   end
 
-  sleep(0.1) unless subject.running?
+  Timeout.timeout(5) do
```

Comment:
I'd prefer we [avoid timeout](http://www.mikeperham.com/2015/05/08/timeout-rubys-most-dangerous-api/).

We should be able to use

`wait(5).for { subject.running? }.to be(true)`